eng/pipelines/templates/jobs/ci.yml

```diff
@@ -336,14 +336,14 @@ jobs:
           arguments: >
             -Directory sdk/${{ parameters.ServiceDirectory }}
 
-      - ${{ if eq(parameters.EnableVerifyTypeSpecCodeGeneration, 'true') }}:
-        - task: PowerShell@2
-          displayName: Verify TypeSpec Code Generation
-          inputs:
-            pwsh: true
-            filePath: $(Build.SourcesDirectory)/eng/scripts/TypeSpec-Compare-CurrentToCodegeneration.ps1
-            arguments: >
-              -Directory sdk/${{ parameters.ServiceDirectory }}
+      - task: PowerShell@2
+        displayName: Verify TypeSpec Code Generation
+        continueOnError: ${{ if eq(parameters.EnableVerifyTypeSpecCodeGeneration, 'false') }}
```

Comment:
Now that this isn't enabling or disabling the verification, we should rename the param to `IgnoreTypeSpecCodeGenVerificationError` or something similar.

Reply:
Renamed! thanks.